stack-lts-2.yaml

```diff
@@ -0,0 +1,12 @@
+resolver: lts-2.22
+packages:
+- '.'
+- location: example/
+  extra-dep: true
```

Comment:
Ah, we disable `example` tests only for ghc 7.8. Okay then, no need to get involved with some trickery.